packages/render-fidelity-tools/src/workflows/update-screenshots/renderer-offline.ts

```diff
@@ -0,0 +1,51 @@
+/* @license
+ * Copyright 2022 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ScenarioConfig } from '../../common.js';
+import { spawnSync } from 'child_process';
+
+export const rendererOffline = async (
+  scenario: ScenarioConfig | null,
+  rendererCommand: string,
+  rendererArgs: string[],
+  outputFile: string): Promise<void> => {
+
+  if (scenario == null) {
+    throw new Error(' Scenario not specified!');
+  }
+
+  if (outputFile == null) {
+    throw new Error(' Output file not specified!');
+  }
+
+  let renderConfig = {
+    scenario: scenario,
+    outputFile: outputFile
+  }
+
+  const args = rendererArgs.concat([JSON.stringify(renderConfig)])
+  const result = spawnSync(rendererCommand, args);
```

Comment:
This looks like a useful stub, but did you actually use it to generate your stellar goldens? I assume you have some code somewhere that does the setup of your renderer based on our config? I guess if no one has access to Stellar, then it doesn't really matter, but it sort of seems like that setup code ought to live our repo somewhere.

Reply:
Yes, I was using it. There's a python render script which sets up stellar based on the scenario config. 
```json
 {
      "name": "stellar",
      "description": "Dassault Systèmes STELLAR",
      "command": {
        "executable": "python",
        "args": ["test/renderers/stellar/stellar.py"]
      }
```

stellar.py is very similar to the existing web renderer integration scripts. 
```python
def main():
  config = json.loads(sys.argv[1])

  scenario = config["scenario"]
  outpath = config["outputFile"]
  scene = loadScene(scenario["model"])
  camera = createCamera(scenario["dimensions"], scenario["orbit"], scenario["verticalFoV"], scenario["target"])
  ibl = loadIBL(scenario["lighting"])
  result = renderScene(scene, camera, ibl)
  saveImage(result, outpath)
```

I don't think it's particular useful to have the full stellar.py in the repo. As you say, stellar is not available anyways, and the stellar specific code obfuscates more than it helps. Maybe a dummy renderer.py? I could add some useful code pieces, e.g. for conversion of camera parameters and the required ACES tonemapping.
Alternatively, a stub like the one above as part of the readme?

Reply:
Yeah, a dummy sounds reasonable if that's not too much work. Can be a follow-on PR.

Reply:
Ok, no problem. I'll open a seperate PR when ready.